internal/config/audit.go

```diff
@@ -93,3 +94,15 @@ type BufferConfig struct {
 	Capacity    int           `json:"capacity,omitempty" mapstructure:"capacity"`
 	FlushPeriod time.Duration `json:"flushPeriod,omitempty" mapstructure:"flush_period"`
 }
+
+type WebhookTemplateConfig struct {
+	Enabled   bool              `json:"enabled,omitempty" mapstructure:"enabled"`
+	Templates []WebhookTemplate `json:"templates,omitempty" mapstructure:"templates"`
+}
+
+type WebhookTemplate struct {
+	Method  string            `json:"method,omitempty" mapstructure:"method"`
```

Comment:
do you think a method other that `POST` would be used? not sure if we need to make methods configurable

Reply:
@markphelps Maybe not, you are right. I made this configurable because was not sure if providers might have endpoints that accept methods other than "POST".